Docs/development/jasp-build-guide-macos.md

```diff
@@ -30,28 +31,44 @@ You can download and install Homebrew using the following command. After running
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
+Note the instructions in mentioned under "Next Steps" after installing homebrew, where `userName` below should be changed to your username
```

Comment:
I have never done this part, and never had any issue with installing Brew. I would not put it in the instruction.

Reply:
I think it's good to add. People tend not to read stuff, for instance, me. Fortunately, I this time read my terminal output carefully.